humble/distribution.yaml

```diff
@@ -1500,6 +1500,48 @@ repositories:
       url: https://github.com/introlab/find-object.git
       version: humble-devel
     status: maintained
+  flexbe_app:
+    doc:
+      type: git
+      url: https://github.com/flexbe/flexbe_app.git
+      version: humble
+   # release:
+   #   packages:
+   #   - flexbe_app
+   #   tags:
+   #     release: release/humble/{package}/{version}
+   #   url: https://github.com/ros2-gbp/flexbe_app-release.git
+   #   version: 3.1.0
```

Comment:
Just remove this section; `bloom` will end up recreating it when you do the actual release.